Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -456,7 +476,10 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
           "res/conditions/sonVolume.png")
       .AddCodeOnlyParameter("currentScene", "")
       .AddParameter("expression", _("Channel identifier"))
-      .UseStandardRelationalOperatorParameters("number")
+      .UseStandardRelationalOperatorParameters(
+          "number",
+          ParameterOptions::MakeNewOptions().SetDescription(
+              _("Volume to compare to (0-100)")))
```

Comment:
Here you've added 0-100 to the description, but not removed it from the label?

Reply:
I think that label and description is the same thing for parameter.